src/Eryph.ConfigModel.Catlets.Json/CatletConfigJsonSerializer.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Eryph.ConfigModel.Catlets;
+
+namespace Eryph.ConfigModel.Json;
+
+public static class CatletConfigJsonSerializer
+{
+    private static readonly Lazy<JsonSerializerOptions> LazyOptions = new(() => 
```

Comment:
The `JsonSerializerOptions` should definitely be a singleton (see https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1869). I just used `Lazy` because it provides a cleaner way initialize a singleton on first access. Additionally, it is completely thread-safe.

Reply:
ok, I was aware of the Singleton requirement, but always assumed that it is better to be static. Ok, keep it that way.